guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using LUKS during host provisioning.
+In addition, you can bind the disk to a Tang server through Clevis for decryption during boot.
```

Comment:
```suggestion
Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
```
Suggesting this because it sounds like an option as well as an addition.

Reply:
Yes, applied.